src/main/java/uk/gov/register/exceptions/FieldConversionException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.register.exceptions;
+
+public class FieldConversionException extends RuntimeException {
```

Comment:
Is it worth renaming this to be `ItemConversionException` and `ItemConversionExceptionMapper`? It might be more consistent with the name `ItemConverter`, but I'm also happy for this to be left as it is.

Reply:
As discussed, happy for this to keep the current name.